src/iop/demosaic.c

```diff
@@ -3917,7 +5210,7 @@ static void demosaic_method_xtrans_callback(GtkWidget *combo, dt_iop_module_t *s
   p->demosaicing_method = dt_bauhaus_combobox_get(combo) | DEMOSAIC_XTRANS;
   if((p->demosaicing_method > (DT_IOP_DEMOSAIC_PASSTHROUGH_MONOCHROME | DEMOSAIC_XTRANS))
      || (p->demosaicing_method < DT_IOP_DEMOSAIC_VNG))
-    p->demosaicing_method = DT_IOP_DEMOSAIC_MARKESTEIJN;
+    p->demosaicing_method = DT_IOP_DEMOSAIC_FDC;
```

Comment:
It needs to be discussed if FDC is to be made default or not as it changes the existing behaviour.

Reply:
I posted that to the previous pr...

Reply:
To be honest, I would also still be conservative at this point and not make FDC the default yet. My impression is also that this must be clipping due to too big method id. I'll investigate and let you know.

Reply:
OK, I corrected this one, too. It wasn't strictly speaking the general default, it was the fallback for invalid values delivered back from the combobox. In any case, Markesteijn is now always default. On the way, I also spotted a minor error in the existing code: ' | DEMOSAIC_XTRANS' was missing once, I think.